src/install/frontend.py

```diff
@@ -17,19 +17,19 @@
 def execute_commands_and_raise_on_return_code(commands, error=None):  # pylint: disable=invalid-name
     for command in commands:
         bad_return = error if error else 'execute {}'.format(command)
-        output, return_code = execute_shell_command_get_return_code(command)
+        output, return_code = execute_shell_command_get_return_code(command, timeout=10)
         if return_code != 0:
             raise InstallationError('Failed to {}\n{}'.format(bad_return, output))
 
 
 def wget_static_web_content(url, target_folder, additional_actions, resource_logging_name=None):
     logging.info('Install static {} content'.format(resource_logging_name if resource_logging_name else url))
     with OperateInDirectory(target_folder):
-        wget_output, wget_code = execute_shell_command_get_return_code('wget -nc {}'.format(url))
+        wget_output, wget_code = execute_shell_command_get_return_code('wget -nc {}'.format(url), timeout=10)
```

Comment:
As again stuff gets downloaded here, the timeout should be less restrictive here. 30 seconds should be enough, though as the downloaded are not that big.